CRM/Utils/Check/Component/Event.php

```diff
@@ -5,6 +5,13 @@
  */
 class CRM_Utils_Check_Component_Event extends CRM_Utils_Check_Component {
 
+  /**
+   * @inheritDoc
+   */
+  public function isEnabled() {
+    CRM_Core_Component::isEnabled('CiviEvent');
```

Comment:
Makes sense - same as the CiviCase one. But needs a `return` statement though.

Reply:
@MegaphoneJon not sure if you saw my comment yesterday - this just needs a `return` in front and then is mergeable.

```suggestion
    return CRM_Core_Component::isEnabled('CiviEvent');
```

Reply:
Thanks, I missed it.  Will fix tomorrow.